### PR TITLE
Make bootstrapping easier for Windows users

### PR DESCRIPTION
**Pull request by saintmatthieu**
- [x] I signed [CLA](https://www.audacityteam.org/cla/)
- [x] The title of the pull request describes an issue it addresses
- [x] If changes are extensive, then there is a sequence of easily reviewable commits
- [x] Each commit's message describes its purpose and effects
- [x] There are no behavior changes unnecessary for the stated purpose of the PR

Recommended:
- [x] Each commit compiles and runs on my machine without known undesirable changes of behavior

### CODE REVIEW

File: au4/BUILDING.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Using Visual Studio
+
+To build Audacity 4 on Windows with Visual Studio rather than Qt Creator, from the root of the repository, run the following command:
+
+```bash
+cmake -S./au4 -B./build4
+```
+
+Open `./build4/audacity.sln`, build `INSTALL`.
+(Couldn't find out how to fix this, but the install still misses jpeg8.dll and libpng16(d).dll. To get them you can build the audacity 3 project and find them in your output dir. Copy them to the install target dir `./build4/install/bin`.)
```
Comment (saintmatthieu) [inline au4/BUILDING.md:10]: Reviewers, if you have an idea how to fix this, please pitch in.
This entire PR isn't meant to be something final and super clean and flawless. It's just one small increment.

Comment (igorkorsukov) [reply au4/BUILDING.md:10]: I'll add them later
(they are not required for the release build on СI, so I didn't notice their necessity when I was setting up the build for Windows)

File: au4/CMakeLists.txt [hunk 2]
```diff
@@ -48,6 +48,7 @@ include(${MUSE_FRAMEWORK_SRC_PATH}/cmake/MuseDeclareOptions.cmake)
 # AudacityDependencies
 ###########################################
 set(AUDACITY_ROOT ${CMAKE_CURRENT_LIST_DIR}/..)
+set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/install")
```
Comment (igorkorsukov) [inline au4/CMakeLists.txt:51]: This parameter is set on the СI, so I think I need to do something like this, it should work
`set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/install" CACHE STRING "Install prefix")`

File: au4/CMakeLists.txt [hunk 3]
```diff
@@ -262,3 +263,7 @@ add_subdirectory(src)
 if (OS_IS_LIN)
     include(${CMAKE_CURRENT_LIST_DIR}/buildscripts/packaging/Linux+BSD/SetupAppImagePackaging.cmake)
 endif(OS_IS_LIN)
+
+if (MSVC)
+   add_compile_options(/wd4100)
```
[the remark this answers is not in the document]
Comment (saintmatthieu) [reply au4/CMakeLists.txt:268]: No that doesn't help. Still getting `warning C4100: 'path': unreferenced formal parameter` and the likes. Will revert this change before merging.

File: au4/BUILDING.md (hunk 1 above)
Comment (igorkorsukov) [reply au4/BUILDING.md:10]: fixed https://github.com/audacity/audacity/pull/6958

**Comment by saintmatthieu** [comment]
Don't know why closing this ticket turned it red like this - it was merged properly.

**Comment by grliszas14** [comment]
@saintmatthieu Windows CI build failed, maybe that's why? just guessing

**Pull request closed without merging**